vagrant.rb

```diff
@@ -185,7 +185,9 @@ def platform(config, pf_id, pf_name)
     # Configure
     config.vm.define name do |cfg|
       # the provisioning script is generated
-      cfg.vm.synced_folder ".", "/vagrant", disabled: true # disable default sync
+      if machine['rudder-setup'] != "dev-server" then
```

Comment:
This should be up by default, use a flag for box who do not need it (aws based box or old os). Kind of like the "sync_folder" that existed before, but using a boolean

Reply:
yes that's what i want, the only usage i found was to transmit a jar for the dev server and there are many box without proper hostfs, that is difficult to track